user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Service_definition.md

```diff
@@ -50,7 +50,7 @@ To configure the component:
 
    - *Visible nodes*: Allows you to select which nodes should be displayed. Below this option, a tree view is displayed, showing the nodes in the service definition. Only the selected nodes will be displayed in the component.
 
-   - *Enable zooming*: Determines whether users will be able to zoom in and out on the graph.
+   - *Enable zooming*: Determines whether users will be able to zoom in and out on the graph. From DataMiner 10.3.0[CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 40017-->, you can zoom in or out by pressing CTRL while using the scroll wheel of your mouse. Prior to DataMiner 10.3.0[CU18]/10.4.0 [CU6]/10.4.9, you can zoom in or out using the scroll wheel alone.
```

Comment:
No instructions were included on how to zoom in/out for the service definition component. I assume zooming in/out used to work by using the scroll wheel on its own instead of pressing CTRL at the same time. Let me know if this isn't correct, @Robin-Debel.

Reply:
Correct.